### DISCUSSION TITLE
Is this mod available for forge

### DISCUSSION THREAD

**Discussion by maxplayz109X** [Virus5600/Defensive-Measures-Mod#4] [Q&A]
I just want to know if this mod is available for forge that will be great it will be available at forge

**Comment by Virus5600** [comment]
There are plans. It's originally planned to be released only on Fabric but after seeing lots of request, I'm considering it. This would however slow my pace as I would have to either re-learn the entire Forge API or refactor the entire project so that it'll match the structure of Architecture API or similar.
